client/src/components/FileStatusIcon/FileStatusIcon.js

```diff
@@ -28,7 +28,7 @@ class FileStatusIcon extends PureComponent {
         'SilverStripe\\Admin\\FileStatusIcon.TRACKED_FORM_UPLOAD_UNRESTRICTED',
         'Form submission, unrestricted access'
       );
-    return { className, 'data-title': dataTitle };
+    return { className, 'data-title': dataTitle, 'aria-label': dataTitle };
```

Comment:
Adds an explicit `aria-label` which both hides the font icon from screen readers and ensures screen readers have the same textual context as sighted users.

Same as the one added below.